docs/content/how-to-guides/installation/install-using-manifests/_index.md

```diff
@@ -0,0 +1,254 @@
+- `--registry <registry>`: Address of the container registry to pull Drasi images from. Default is "ghcr.io".
+- `--version <tag>`: Container image version tag to use. Default is the CLI version.
+- `--observability-level <level>`: Specifies observability infrastructure (`none`, `tracing`, `metrics`, `full`).
+
+Example with custom settings:
+
+```bash
+drasi init --manifest ./drasi-manifests --namespace drasi-prod --version 0.1.5 --observability-level metrics
+```
+
+## Install Drasi Using the Generated Manifests
+
+Follow these steps to install Drasi using the generated manifests:
+
+### Step 1: Verify Dapr Installation
+
+Ensure Dapr is installed in your cluster:
+
+```bash
+kubectl get pods -n dapr-system
+```
+
+You should see output similar to:
+
+```
+NAME                                     READY   STATUS    RESTARTS   AGE
+dapr-dashboard-5cc65d985f-qzqbg          1/1     Running   0          10m
+dapr-operator-5d98f57c86-kspwk           1/1     Running   0          10m
+dapr-placement-server-0                  1/1     Running   0          10m
+dapr-sentry-697bdc6cc4-xprww             1/1     Running   0          10m
+dapr-sidecar-injector-56c4c4b485-n48bg   1/1     Running   0          10m
+```
+
+If Dapr is not installed, follow the [Dapr installation guide](https://docs.dapr.io/getting-started/install-dapr-kubernetes/).
+
+### Step 2: Apply the Infrastructure Manifest
+
+Apply the Kubernetes infrastructure manifest first:
+
+```bash
+kubectl apply -f kubernetes-resources.yaml
+```
+
+This will create:
+- The Drasi namespace (if it doesn't exist)
+- Infrastructure components
+- Required ConfigMaps and Secrets
+- RBAC resources
+- Drasi control plane
+
+Wait for the infrastructure components to be ready:
+
+```bash
+kubectl wait --for=condition=available --timeout=300s deployment/drasi-api -n drasi-system
+```
+
+### Step 3: Configure Drasi CLI Environment
+
+Configure the Drasi CLI to point to your current Kubernetes context:
+
+```bash
+drasi env kube
+```
+
+This command will:
+- Add your current Kubernetes context as a Drasi environment
+- Set it as the active environment for Drasi CLI commands
+
+### Step 4: Apply the Drasi Resources Manifest
+
+Apply the Drasi-specific resources:
+
+```bash
+drasi apply -f drasi-resources.yaml
+```
+
+This will create:
+- Control plane components (API server, resource provider)
+- Default query container
+- Source providers (PostgreSQL, SQL Server, Cosmos Gremlin, etc.)
+- Reaction providers (Debug, SignalR, EventGrid, etc.)
+
+### Step 5: Verify the Installation
+
+Check that all Drasi components are running:
+
+```bash
+drasi list querycontainer
+```
+
+You should see output similar to:
+
+```
+    ID    | AVAILABLE
+----------+------------
+  default | true
+```
+
+## Example: Complete Installation Workflow
+
+Here's a complete example of installing Drasi using manifests:
+
+```bash
+# 1. Generate manifests in a specific directory
```

Comment:
If `drasi init --manifest` does not install Dapr, should the first step be to ask users to install Dapr?

Reply:
Or should we configure this flag to install Dapr as usual, and then return the resource yaml files?